rm/rm-server/src/test/java/functionaltests/RegressionTestSuite.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package functionaltests;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+import functionaltests.nodesource.TestSSHInfrastructureV2RestartDownNodesPolicy;
+import functionaltests.nodesrecovery.RecoverLocalInfrastructureTest;
+import functionaltests.nodesrecovery.RecoverSSHInfrastructureV2Test;
+import functionaltests.nonblockingcore.NonBlockingCoreTest;
+import functionaltests.permissions.TestNSNodesPermissions;
+import functionaltests.selectionscript.SelectionScriptTimeOutTest;
+
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({ NonBlockingCoreTest.class, TestNSNodesPermissions.class, SelectionScriptTimeOutTest.class,
+                      RecoverLocalInfrastructureTest.class, RecoverSSHInfrastructureV2Test.class,
+                      TestSSHInfrastructureV2RestartDownNodesPolicy.class })
```

Comment:
I would add here the `TestLocalInfrastructureRestartDownNodesPolicy.class` as well.

Reply:
It could be, but it is not so unstable, and leaving it as a standard test allows at least to test the restartDownNode policy outside of regression